virttest/test_setup.py

```diff
@@ -818,15 +818,15 @@ def _assemble_iptables_rules(self, port_list):
         return rules
 
     def _add_bridge(self):
-        process.system("brctl addbr %s" % self.brname)
+        addbr_cmd = ("ip link add name %s type bridge stp_state 1"
+                     " forward_delay 400" % self.brname)
+        process.system(addbr_cmd)
         ip_fwd_path = "/proc/sys/net/%s/ip_forward" % self.ip_version
-        ip_fwd = open(ip_fwd_path, "w")
-        ip_fwd.write("1\n")
-        process.system("brctl stp %s on" % self.brname)
-        process.system("brctl setfd %s 4" % self.brname)
+        with open(ip_fwd_path, "w") as ip_fwd:
```

Comment:
sound like original logic order is firstly change value in ip_fwd_path, then enable bridge.
After you change, it sound like you already enable bridge, then change value in ip_fwd_path.
 please confirm whether it impact behaviour somehow.